### PR TITLE
Christopher Beards

### PR DESCRIPTION
**Pull request by ChristopherBeards**
## Trello Set Up:
* [x] Create a Trello account
* [x] Create a new board called "Lambda Notes - {Your Name}"
* [x] Create lists titled `backlog`,`To Do`, `In Progress`, and `Done`
* [x] Fill in the `To Do` list with the MVP features listed below
* [x] Fill in the `backlog` list with all the extra features listed below
* [x] Share your board with the Project Manager that has been assigned to you.  If you have not been assigned yet, reach out to your lead TA for guidance
* [x] Add your Trello URL to your project's README.md file. Commit the change, push it to your repository & submit a pull request

## MVP Features:
* [x] Display a list of notes
* [x] Create a note with a title and content
* [x] View an existing note
* [x] Edit an existing note
* [x] Delete an existing note

### CODE REVIEW

**Comment by Mjax511** [comment]
## MVP Features:
* [ ] Display a list of notes
* [ ] Create a note with a title and content
* [ ] View an existing note
* [ ] Edit an existing note
* [ ] Delete an existing note

**Comment by mixelpixel** [comment]
@ChristopherBeards & @Mjax511 I just went ahead and put the MVP task list in the _**first**_ PR comment, now it displays inthe PR list:

@ChristopherBeards would you be so kind to check off the appropriate boxes? 

Looks like you a killing it! I love the Git commit messages with titles and bodies 🏆

**Comment by mixelpixel** [comment]
I added the Trello Set Up Task list to the first PR comment.

::googles Lodash::

**Comment by ChristopherBeards** [comment]
Project Complete; before Back-End-Project Week.

File: lambda-notes/src/styles/ViewNotes.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.noteBody {
+  font-size: 14px;
+  color: #4a494a;
+  letter-spacing: 1px;
+  line-height: 29px;
+  height: 150px;
+  margin-top: 7px;
+  overflow: hidden;
```
Comment (Velsu) [inline lambda-notes/src/styles/ViewNotes.css]: Nice one, thanks to this body text won't overflow. It is also possible to just `trim` the text after certain length.

File: lambda-notes/src/components/EditNote.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                  value={this.state.note.title}
+                  name="title"
+                  onChange={this.updateState}
+                />
+              </div>
+              <div className="bodyContainer">
+                <textarea
+                  type="text"
+                  className="bodyField"
+                  value={this.state.note.body}
```
Comment (Velsu) [inline lambda-notes/src/components/EditNote.js]: Many people forget that edit requires actual value present and not placeholder, good job on that.

File: lambda-notes/src/App.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import ViewNotes from './components/ViewNotes';
+import DisplayOne from './components/DisplayOne';
+import CreateNote from './components/CreateNote';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <div>
```
Comment (Velsu) [inline lambda-notes/src/App.js]: I like how `App.js` is so clean and focuses on routing.

**Pull request closed without merging**